common/src/app/common/pages/helpers.cljc

```diff
@@ -496,3 +501,9 @@
   (and (not= (:type shape) :frame)
        (= (:frame-id shape) uuid/zero)))
 
+(defn children-seq
+  "Creates a sequence of shapes through the objects tree"
+  [shape objects]
+  (tree-seq #(d/not-empty? (get shape :shapes))
+            #(->> (get % :shapes) (map (partial get objects)))
```

Comment:
I very minor improvement, take the `(partial get objects)` on a let binding  before `tree-seq` (for avoid creating the instance all the time is executed and  define it as `#(get objects %)` instead of using partial. Partial has performance overhead.

```
Execution time sample mean : 9.721077 ns (for partial)
Execution time sample mean : 3.744206 ns (for the reader macro)
```

Reply:
ok